tests/src/ExistingSiteJavascript/EntityCloneIntegrationTest.php

```diff
@@ -31,18 +29,21 @@ public function testClone(): void {
 
     $this->drupalGet($edit_form);
     $page = $this->getSession()->getPage();
-    $assert_session = $this->assertSession();
-
+    // $assert_session = $this->assertSession();
     // Add Hero component to the layout canvas.
     $this->getLayoutCanvas()->add('Hero');
     $page->pressButton('Save');
 
+    // @todo The action buttons disappeared on the node edit page, isue raised
```

Comment:
@rajeshreeputra We can add conditional check to skip this testmethod by using  $this->markTestSkipped
Or apply any better approach

cc: @vishalkhode1

Reply:
The issue with gin theme is now fixed in dev release, hence reverted this temp. commit.